core/templates/dev/head/services/SpeechSynthesisChunkerService.ts

```diff
@@ -170,21 +170,22 @@ export class SpeechSynthesisChunkerService {
     var elt = $('<div>' + html + '</div>');
     // Convert links into speakable text by extracting the readable value.
     elt.find('oppia-noninteractive-' + this.RTE_COMPONENT_NAMES.Link)
-      .replaceWith(() => {
+      .replaceWith(function() {
         if ((<HTMLElement><any> this).attributes[
           'text-with-value'] !== undefined) {
           return (<HTMLElement><any> this).attributes[
             'text-with-value'].textContent.replace(/&quot;/g, '');
         }
       });
 
+    var _this = this;
```

Comment:
@kevintab95 Should I add a comment in the file to explain this code or it's obvious?